ci-operator/step-registry/openshift-extended/test/openshift-extended-test-commands.sh

```diff
@@ -306,17 +306,21 @@ function run {
     fi
     # summarize test results
     echo "Summarizing test result..."
-    mapfile -t test_suite_failures < <(grep -r -E 'testsuite.*failures="[1-9][0-9]*"' "${ARTIFACT_DIR}" | grep -o -E 'failures="[0-9]+"' | sed -E 's/failures="([0-9]+)"/\1/')
-    failures=0
-    for (( i=0; i<${#test_suite_failures[@]}; ++i ))
-    do
-        let failures+=${test_suite_failures[$i]}
-    done
-    if [ $((failures)) == 0 ]; then
-        echo "All tests have passed"
-    else
-        echo "${failures} failures in openshift-extended-test" | tee -a "${SHARED_DIR}/openshift-e2e-test-qe-report-openshift-extended-test-failures"
+    failures=0 errors=0 skipped=0 tests=0
```

Comment:
@liangxia please also change it for golang step openshift-extended-test-disruptive and openshift-extended-test-longduration. thanks